src/geometry/transform.rs

```diff
@@ -452,6 +454,53 @@ where DefaultAllocator: Allocator<N, DimNameSum<D, U1>, DimNameSum<D, U1>>
     }
 }
 
+impl<N, D: DimNameAdd<U1>, C> Transform<N, D, C>
+where
+    N: Real,
+    C: TCategory,
+    DefaultAllocator: Allocator<N, DimNameSum<D, U1>, DimNameSum<D, U1>>
+        + Allocator<N, DimNameSum<D, U1>>
+        + Allocator<N, D, D>
+        + Allocator<N, D>,
+{
+    /// Transform the given point by this transformation.
```

Comment:
I didn't add doctests to the `Transform` methods since I thought they might be too verbose to be helpful, but I can add them if you'd like.

Reply:
No doc-test here is fine for now.